crates/ldap/src/core/user.rs

```diff
@@ -21,6 +21,25 @@ use lldap_domain_handlers::handler::{UserListerBackendHandler, UserRequestFilter
 use lldap_domain_model::model::UserColumn;
 use tracing::{debug, instrument, warn};
 
+pub const REQUIRED_USER_ATTRIBUTES: &[&str] = &["user_id", "mail"];
+
+const DEFAULT_USER_OBJECT_CLASSES: &[&str] =
```

Comment:
Can you use that in `convert_user_filter` as well? Similarly for groups. There might be some casing to handle.